templates/experiments/experiment_form.html

```diff
@@ -122,6 +122,19 @@ <h3 class="text-md font-bold">{% translate "Please note" %}</h3>
       </div>
     </div>
 {% endblock form %}
+{% block form_actions %}
+  <div class="mt-2">
+    <input type="submit" class="pg-button-primary" value="{{ button_text }}">
+    {% if experiment.id %}
+      {% flag "experiment_versions" %}
+        <button type="submit" class="pg-button-secondary" name="action" {% if disable_version_button %}disabled="True"{% endif %} value="save_and_version">Create Version</button>
+      {% endflag %}
+      <a href="{% url 'experiments:delete' team.slug experiment.id %}" class="pg-button-danger" onclick="return confirm('Are you sure you want to delete this experiment? This action cannot be undone.')">
```

Comment:
@stephherbers this doesn't redirect after clicking it - it should go back to the experiment list page